x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_single_domain_logic.test.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  LogicMounter,
+  mockHttpValues,
+  mockFlashMessageHelpers,
+} from '../../../__mocks__/kea_logic';
+import '../../__mocks__/engine_logic.mock';
+
+import { nextTick } from '@kbn/test/jest';
+
+import { CrawlerSingleDomainLogic, CrawlerSingleDomainValues } from './crawler_single_domain_logic';
+import { CrawlerDomain } from './types';
+
+const DEFAULT_VALUES: CrawlerSingleDomainValues = {
+  dataLoading: true,
+  domain: null,
+};
+
+describe('CrawlerSingleDomainLogic', () => {
+  const { mount } = new LogicMounter(CrawlerSingleDomainLogic);
+  const { http } = mockHttpValues;
+  const { flashAPIErrors } = mockFlashMessageHelpers;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mount();
+  });
+
+  it('has expected default values', () => {
+    expect(CrawlerSingleDomainLogic.values).toEqual(DEFAULT_VALUES);
+  });
+
+  describe('actions', () => {
+    describe('onReceiveDomainData', () => {
+      const domain = {
+        id: '507f1f77bcf86cd799439011',
+      };
+
+      beforeEach(() => {
+        CrawlerSingleDomainLogic.actions.onReceiveDomainData(domain as CrawlerDomain);
+      });
+
+      it('should set the crawl requests', () => {
```

Comment:
Should this have been

```suggestion
      it('should set the domain', () => {
```

?